packages/browser/src/integrations/globalhandlers.ts

```diff
@@ -129,4 +129,48 @@ export class GlobalHandlers implements Integration {
 
     return event;
   }
+
+  private _eventFromIncompleteRejection(stacktrace: TraceKitStackTrace, error: any): Event {
+    const event: Event = {
+      level: Severity.Error,
+    };
+
+    if (isPrimitive(error)) {
+      event.exception = {
+        values: [
+          {
+            type: 'UnhandledRejection',
+            value: `Non-Error promise rejection captured with value: ${error}`,
+          },
+        ],
+      };
+    } else {
+      event.exception = {
+        values: [
+          {
+            type: 'UnhandledRejection',
+            value: `Non-Error promise rejection captured with keys: ${keysToEventMessage(Object.keys(error).sort())}`,
+          },
+        ],
+      };
+      event.extra = {
+        __serialized__: normalizeToSize(error),
+      };
+    }
+
+    if (event.exception.values && event.exception.values[0]) {
+      event.exception.values[0].mechanism = {
+        data: {
+          mode: stacktrace.mode,
+          incomplete: true,
+          ...(stacktrace.message && { message: stacktrace.message }),
+          ...(stacktrace.name && { message: stacktrace.name }),
```

Comment:
```suggestion
          ...(stacktrace.name && { name: stacktrace.name }),
```

Thanks @lobsterkatie :D